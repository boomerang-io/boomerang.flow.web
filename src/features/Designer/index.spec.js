import React from "react";
import { Route } from "react-router-dom";
import { WorkflowManagerContainer as WorkflowManager } from "./index";

jest.mock("@boomerang/carbon-addons-boomerang-react", () => ({
  NoDisplay: "NoDisplay",
  LoadingAnimation: "LoadingAnimation",
  notify: "notify",
  Notification: "Notification"
}));

const mockfn = jest.fn();

const props = {
  tasks: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  teams: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  workflow: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  workflowRevision: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  }
};

describe("WorkflowManager --- Snapshot", () => {
  it("Capturing Snapshot of WorkflowManager", () => {
    const { baseElement } = rtlReduxRouterRender(
      <Route path="/editor/:workflowId">
        <WorkflowManager {...props} />
      </Route>,
      { path: "/editor/1234" }
    );
    expect(baseElement).toMatchSnapshot();
  });
});
