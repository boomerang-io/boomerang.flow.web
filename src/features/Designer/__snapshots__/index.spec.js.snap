// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkflowManager --- Snapshot Capturing Snapshot of WorkflowManager 1`] = `
<body
  id="app"
>
  <div />
</body>
`;
